astropy/cosmology/core.py

```diff
@@ -99,13 +99,13 @@ class FLRW(Cosmology):
         Neff. Usually this means you must provide three neutrino masses unless
         you are considering something like a sterile neutrino.
 
-    name : str
-        Optional name for this cosmological object.
-
-    Ob0 : float
+    Ob0 : float or None
```

Comment:
More to the point, I think rather than "float or none" this should just be "float, optional".  Which is to say if you pass in a value for this argument it had better be a float, but specifying it is optional.  Same for the similar changes below.